src/styles/helpers/_colors.scss

```diff
@@ -0,0 +1,10 @@
+@import "../core/settings";
+@import "../core/functions";
+
+$ns: ns();
```

Comment:
sorry, this line doesn't say anything to me from the naming prospect of view. Some comments are appreciated